rules/redis.mk

```diff
@@ -6,7 +6,11 @@ REDIS_TOOLS = redis-tools_$(REDIS_VERSION)_$(CONFIGURED_ARCH).deb
 $(REDIS_TOOLS)_SRC_PATH = $(SRC_PATH)/redis
 $(REDIS_TOOLS)_DEPENDS += $(LIBHIREDIS_DEV)
 $(REDIS_TOOLS)_RDEPENDS += $(LIBHIREDIS)
+# TODO: docker-sonic-p4 depends on redis-tools in Jessie.
+#       Remove this file and src/redis after that resolved.
+ifneq ($(BLDENV),buster)
```

Comment:
What about Stretch? The comment says "docker-sonic-p4 depends on redis-tools in Jessie", but the conditional only checks for Buster. Either update the comment or the conditional.